### PR TITLE
Undefined Object#timeout for issue#351

### PR DESCRIPTION
**Pull request by inokappa**
Hi,

I tried undefine Object#timeout for #351.
But, I'm not sure if this is correct... :weary: 
Please confirm. :pray: :pray: :pray: :pray: :pray: 

Regards.

Yohei

### CODE REVIEW

File: lib/awspec/ext/object.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Object
+  undef :timeout
```
Comment (k1LoW) [inline lib/awspec/ext/object.rb]: Maybe you set `undef :timeout` to [Awspec::Type::Base](https://github.com/k1LoW/awspec/blob/master/lib/awspec/type/base.rb) .

File: spec/type/elasticache_cache_parameter_group_spec.rb [hunk 2]
```diff
@@ -5,4 +5,10 @@
   it { should exist }
   its(:activerehashing) { should eq 'yes' }
   its(:client_output_buffer_limit_pubsub_hard_limit) { should eq '33554432' }
+  its(:repl_timeout) { should eq '60' }
+  its(:timeout) { should eq '0' }
```
Comment (k1LoW) [inline spec/type/elasticache_cache_parameter_group_spec.rb]: 👍

File: lib/awspec/ext/object.rb (hunk 1 above)
Comment (inokappa) [reply lib/awspec/ext/object.rb]: @k1LoW 

Thank you! :bowing_man: 
I moved `undef :timeout` to lib/awspec/ext/object.rb.

Pleaes confirm.

File: lib/awspec/type/base.rb [hunk 3]
```diff
@@ -43,5 +43,7 @@ def method_missing(name)
         method_missing_via_black_list(name, delegate_to: resource)
       end
     end
+
+    undef :timeout
```
Comment (k1LoW) [inline lib/awspec/type/base.rb]: ⭐️

**Comment by k1LoW** [comment]
Thank you @inokappa !!! LGTM !!

**Comment by k1LoW** [comment]
Released as v1.4.2.

**Comment by k1LoW** [comment]
Released as v1.5.3.

**Pull request merged**